src/analysis/processing/qgsalgorithmremovenullgeometry.cpp

```diff
@@ -60,7 +61,10 @@ QString QgsRemoveNullGeometryAlgorithm::shortHelpString() const
 {
   return QObject::tr( "This algorithm removes any features which do not have a geometry from a vector layer. "
                       "All other features will be copied unchanged.\n\n"
-                      "Optionally, the features with null geometries can be saved to a separate output." );
+                      "Optionally, the features with null geometries can be saved to a separate output.\n\n"
+                      "If 'Also remove empty geometries' is checked, the algorithm removes features whose geometries"
```

Comment:
```suggestion
                      "If 'Also remove empty geometries' is checked, the algorithm removes features whose geometries "
```

Reply:
Applied in https://github.com/qgis/QGIS/pull/35084/commits/3b6c04f70e57c2ce3b6bb2bee93c3f643f7a88b6